src/OpenTK/Input/GamePadConfiguration.cs

```diff
@@ -189,25 +189,12 @@ static GamePadConfigurationSource ParseSource(string item)
         /// - # is a zero-based integer number
         /// </summary>
         /// <param name="item">The string to parse</param>
-        /// <returns>The index of the axis</returns>
-        static int ParseAxis(string item)
+        /// <returns>The index of the axis / button</returns>
```

Comment:
Bit of a nitpick, but avoid using contracting characters like this in documentation. It's fine if it's verbose, as long as it's perfectly clear and maintains a direct tone. "axis or button" is preferable.